src/monodroid/jni/dylib-mono.h

```diff
@@ -137,6 +137,10 @@ inline MonoCounters& operator |= (MonoCounters& left, MonoCounters right)
 	typedef struct _MonoProfiler {} MonoProfiler;
 #endif
 
+#ifndef MonoProfilerHandle
```

Comment:
This apparently no longer works with modern compilers. It used to be that the preprocessor "knew" the defined structs/symbols/etc, but this is no longer true. It's better to just risk a warning and redefine the `typedef` here. In C++ mode it might be better to use the `using MonoProfilerHandle = _MonoProfilerDesc*` definition.

Reply:
I guess it can stay that way so that it doesn't change style here and leave it to time when we decide to change it for this whole part.

Reply:
Sure, it's not a biggie - just a remark :)